app/src/main/res/layout/fragment_edit_profile.xml

```diff
@@ -65,7 +65,7 @@
                 style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense"
                 android:layout_width="@dimen/layout_margin_none"
                 android:layout_height="wrap_content"
-                android:layout_marginLeft="@dimen/layout_margin_small"
+                android:layout_marginStart="@dimen/layout_margin_small"
```

Comment:
Add marginLeft back. Although minSdkVersion has been upgraded to 21, @iamareebjamal has said we should be able to revert minSdkVersion back to 16 if possible

Reply:
resolved